### PR TITLE
(TK-490,TK-491) Use tk-status for additional memory info

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

**Comment by Magisus** [comment]
Does this need to go out right away?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ## [Unreleased]
+- update tk-status to 1.1.1 which includes additional java memory info
 
 ## [1.7.31]
```
Comment (Magisus) [inline CHANGELOG.md:4]: I guess the switch from master to 1.7.x did something weird here (most recent version should be 1.7.38), but I guess it's fine.

Comment (justinstoller) [reply CHANGELOG.md:4]: Whoops, I'll fiddle with it and see if I can get it right.

**Comment by justinstoller** [comment]
> Does this need to go out right away?

It's not urgent, but it should go out in the next releases though.

**Pull request merged**